schemas/context.json

```diff
@@ -384,6 +384,12 @@
 				"nullable"
 			],
 			"defaultLocaleKey": "default.submission.step.reviewerSuggestions"
+		},
+		"continuousPublication": {
```

Comment:
I think rather than naming the workflow ("continuous publication") we should describe the entity. In this case, what `continuousPublication` means is that the submission is published, regardless of its status being scheduled or not against an issue. To match the `issues` table, we should do this by adding a `published` column to the `publications` table and use it instead. A first-class column will perform better (and reduce the joins), and it'll be less confusing than a `continuousPublication` setting.